tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -455,28 +456,47 @@ public function makeDavRequest(
 	 * @param string|null $height
 	 *
 	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
 	 */
 	public function downloadPreviews(string $user, ?string $path, ?string $doDavRequestAsUser, ?string $width, ?string $height):void {
-		$user = $this->getActualUsername($user);
-		$doDavRequestAsUser = $this->getActualUsername($doDavRequestAsUser);
 		$urlParameter = [
 			'x' => $width,
 			'y' => $height,
 			'forceIcon' => '0',
 			'preview' => '1'
 		];
-		$this->response = $this->makeDavRequest(
+		$davVersion = $this->getDavPathVersion();
+		if ($davVersion === WebDavHelper::DAV_VERSION_SPACES) {
+			$fileId = urlencode(strstr($this->getFileIdForPath($doDavRequestAsUser ?? $user, $path), '!', true));
+
+			$eTag = str_replace('"', '', $this->getFileEtagForPath($doDavRequestAsUser ?? $user, $path));
+			$url = $this->getBaseUrl() . '/remote.php/dav/spaces/' . $fileId . '/';
+
+			$urlParameter = [
+				'scalingup' => 0,
+				'preview' => '1',
+				'a' => '1',
+				'c' => $eTag,
```

Comment:
do we really need etag?

Reply:
from my finding and inspecting the browser, we need `etag` but if there is an alternative that I am unaware of. Please share with me.

Reply:
if it does not work without the etag, we need to find out if that is a bug or feature

Reply:
viktor is looking over it

Reply:
@saw-jan We are going to use `etag` confirmed by owncloud side reason mention in the description

Reply:
so, use it for others too (old and new)